packages/amplify-graphql-api-construct-tests/src/__tests__/backends/configurable-stack/app.ts

```diff
@@ -62,15 +69,14 @@ const userPoolClient = userPool.addClient(`${PREFIX}UserPoolClient`, {
   },
 });
 
-const testDefinitionDir = path.normalize(path.join(__dirname, '..'));
-const combinedDefinition = combineTestDefinitionsInDirectory(testDefinitionDir);
+const combinedDefinition = combineTestDefinitionsInDirectory(projRoot);
 
 new AmplifyGraphqlApi(stack, `${PREFIX}Api`, {
   definition: combinedDefinition,
   authorizationModes: {
     defaultAuthorizationMode: 'AMAZON_COGNITO_USER_POOLS',
     userPoolConfig: { userPool },
-    apiKeyConfig: { expires: Duration.days(360) },
+    apiKeyConfig: { expires: Duration.days(7) },
```

Comment:
Since these are test apps meant to be torn down, why not go with the minimum i.e 1 day?

Reply:
Reasonable. Will fix in upcoming PR